src/LettuceEncrypt/LettuceEncryptKestrelHttpsOptionsExtensions.cs

```diff
@@ -39,7 +39,7 @@ public static class LettuceEncryptKestrelHttpsOptionsExtensions
                 throw new InvalidOperationException(MissingServicesMessage);
             }
 
-#if NETCOREAPP3_0
+#if NETCOREAPP3_0 || NETCOREAPP3_1
```

Comment:
https://github.com/natemcmaster/LettuceEncrypt/blob/2e5b5bf7dd5ad57c29ca94d37c230050398ee7fe/src/LettuceEncrypt/LettuceEncrypt.csproj#L4
That doesn't do anything unless you change the project TFMs (which is unnessiary, the 3.0 binary can be used on 3.1)

Reply:
3.1 already works.